### PR TITLE
fix(edge-api/tasks): Add change_by_user_id

### PR DESCRIPTION
**Pull request by gagantrivedi**
Thanks for submitting a PR! Please check the boxes below:

- [ ] I have added information to `docs/` if required so people know about the feature!
- [x] I have filled in the "Changes" section below?
- [x] I have filled in the "How did you test this code" section below?
- [x] I have used a [Conventional Commit](https://www.conventionalcommits.org/en/v1.0.0/) title for this Pull Request

## Changes

Add change_by_user_id to avoid old tasks from crashing
related: https://github.com/Flagsmith/flagsmith/pull/4561/files

## How did you test this code?

Adds a unit test

### CODE REVIEW

File: api/tests/unit/edge_api/identities/test_unit_edge_api_identities_tasks.py [hunk 1]
```diff
@@ -128,6 +128,50 @@ def test_call_environment_webhook_for_feature_state_change_with_previous_state_o
     assert data["timestamp"] == now_isoformat
 
 
+def test_call_environment_webhook_for_feature_state_change_with_changed_by_user_id(
+    mocker, environment, feature, identity, admin_user
+):
```
Comment (matthewelwell) [inline api/tests/unit/edge_api/identities/test_unit_edge_api_identities_tasks.py]: Typing please

**Review by matthewelwell — COMMENTED** [review]
Minor comment re: typing

File: api/tests/unit/edge_api/identities/test_unit_edge_api_identities_tasks.py (hunk 1 above)
Comment (gagantrivedi) [reply api/tests/unit/edge_api/identities/test_unit_edge_api_identities_tasks.py]: Ahh, we should make pre-commit catch this

File: api/tests/unit/edge_api/identities/test_unit_edge_api_identities_tasks.py [hunk 2]
```diff
@@ -128,6 +131,54 @@ def test_call_environment_webhook_for_feature_state_change_with_previous_state_o
     assert data["timestamp"] == now_isoformat
 
 
+def test_call_environment_webhook_for_feature_state_change_with_changed_by_user_id(
+    mocker: MockerFixture,
+    environment: Environment,
+    feature: Feature,
+    identity: Identity,
+    admin_user: FFAdminUser,
+):
```
Comment (matthewelwell) [inline api/tests/unit/edge_api/identities/test_unit_edge_api_identities_tasks.py]:
```suggestion
) -> None:
```

**Review by matthewelwell — APPROVED** [review]

**Pull request merged**